### DISCUSSION TITLE
how to lower speed

### DISCUSSION THREAD

**Discussion by omega3** [rhasspy/piper#267] [Q&A] [answered]
I have found this [topic](how can I control the speed? · Issue #249 · rhasspy/piper
https://github.com/rhasspy/piper/issues/249) but to me with these the settings text sounds ridiculous. 

Is there a build-in way to control speed? I need to record mp3, so just playing with different speed is not a proper solution for me.

**Comment by synesthesiam** [comment]
You can change the speed with `--length-scale <N>` where `N > 1` will be slower. Another option is to use [sonic](https://github.com/waywardgeek/sonic)

**Reply by omega3** [reply]
Thank you very much. Can you also explain what --noise-scale and --noise-w are? Maybe you can write it in readme.md.
Which noise to use to record louder and what values it accepts, % or 1,5...

**Reply by synesthesiam** [reply, marked as answer]
They are between 0 and 1. Noise scale increasing the variation in speaking, while noise w changes the variance in phoneme length. It can change depending on the voice, so it's best to try out different values and hear for yourself. Setting both to 0 will sound pretty monotone.
